tracer/build/_build/docker/centos7.dockerfile

```diff
@@ -85,8 +85,10 @@ ENV CC=clang
 FROM base as builder
 
 # Copy the build project in and build it
+COPY *.csproj *.props *.targets /build/
+RUN dotnet restore /build
 COPY . /build
-RUN dotnet build /build
+RUN dotnet build /build --no-restore
```

Comment:
😭 FYI, we don't cache between builds... so this will actually slow down the build compared to the previous (more layers) unless you also go through and rebuild all the VMSS vms in Azure and AWS...😬 which you're welcome to do of course! 😄

Reply:
Good to know. I'll keep this change local-only, then. It was shaving off ~20 seconds on each local build by not restoring every time.

Reply:
Yeah, I agree this is a good change to make, just be aware that there's more involved 😄

Reply:
FYI, I incorporated this suggestion in the dockerfiles when I did the .NET 8 update